### PR TITLE
Actioning feedback on ADR's

### PR DESCRIPTION
**Pull request by danielgrant**

### CODE REVIEW

**Review by stephenwf — APPROVED** [review]

File: docs/adr/0001-choice-of-rdbms.md [hunk 1]
```diff
@@ -1,75 +1,104 @@
...
-* PostgreSQL
-* MySQL
+* PostgreSQL (https://www.postgresql.org/)
+* MySQL (https://www.mysql.com/products/community/)
 
 ## Decision Outcome
 
-Chosen option: MySQL, because BFI's experience with their existing MySQL databases will make maintenance and reporting easier for them in the long run.
-Additionally, the data model we will be creating is simple enough that we wouldn't expect to put any features of the database to its limits.
+MySQL is selected as the solution as it aligns provides BFI with an RDMS
```
Comment (pmollahan) [inline docs/adr/0001-choice-of-rdbms.md:30]: Remove 'aligns'

File: docs/adr/0002-choice-of-auth-provider.md [hunk 2]
```diff
@@ -34,13 +35,62 @@ either.
...
+* Auth0 hosted UI's can be customised but only to a limited extent.
+* Limit of 7000 monthly active users.
+
+## Pros and Cons of the Options
+
+### Auth0
+
+#### Pros
+
+* Fffers a free tier with basic features, with the option to upgrade /
```
Comment (pmollahan) [inline docs/adr/0002-choice-of-auth-provider.md:62]: 'Offers'

File: docs/adr/0003-choice-of-technology-and-framework.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+* Kotlin (https://kotlinlang.org/) with Spring (https://spring.io/)
+* TypeScript (https://www.typescriptlang.org/) with Express.js
+  (https://expressjs.com/)
+* Python (https://www.python.org/) with Django
+  (https://www.djangoproject.com/)
+
+## Decision Outcome
+
+Kotlin is selected as the solution owing to Digirati's vast experience
+with the JVM, and when combined with Spring will make this the
```
Comment (pmollahan) [inline docs/adr/0003-choice-of-technology-and-framework.md:39]: Remove superfluous 'this'

File: docs/adr/0003-choice-of-technology-and-framework.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+deliverables straightforward.
+
+Kotlin does not have the boilerplate cost associated with Java, and as a
+result internal there should be fewer barriers in identifying and
```
Comment (pmollahan) [inline docs/adr/0003-choice-of-technology-and-framework.md:43]: 'internal' - is this relevant or perhaps requires expansion?

**Review by pmollahan — APPROVED** [review]

**Pull request merged**